usernames_creator.py

```diff
@@ -0,0 +1,7 @@
+# user_100000 -> user_100500
```

Comment:
Is this file really useful? It's just a for without any public API

Reply:
ahh this is just for generating usernames for spraying. It can be taken as an additional tool

Reply:
I see your point, but this file does not export anything. Could you please add at least some structure for this code to be called easily?